src/rgw/rgw_sal_daos.cc

```diff
@@ -1950,9 +1981,13 @@ DaosAtomicWriter::DaosAtomicWriter(
       ptail_placement_rule(_ptail_placement_rule),
       olh_epoch(_olh_epoch),
       unique_tag(_unique_tag),
-      obj(_store, _head_obj->get_key(), _head_obj->get_bucket()) {}
+      obj(_store, _head_obj->get_key(), _head_obj->get_bucket())
+{
+  ldpp_dout(dpp, 20) << "DEBUG: DaosAtomicWriter::DaosAtomicWriter" << dendl;
+}
```

Comment:
This is just a constructor, is it necessary?

Reply:
you're right.  I guess I got too happy with adding those